gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+- A [Subgraph](https://github.com/graphprotocol/block-oracle/tree/main/packages/subgraph) that indexes the Data Edge allows indexers to craft the POI easily.
+
+### Current design
+
+The current system is run by a single Operator, which is the largest trust assumption of the EBO.
+
+![EBO Designs (18).png](../assets//gip-0080/Optimistic%20EBO%20Designs.png)
+
+### The Risks
+
+- **Single Operator risk:** The current system is run by a single operator, which is the most prominent trust assumption of the EBO. There have been discussions around increasing the committee of validators/proposers.
+- **Liveness risks:** If the owner fails to update the EBO, indexers won’t have a reference block to build their POIs, preventing them from closing their allocations or risk slashing if they register a POI for an incorrect block. The system must be redundant to mitigate this risk.
+- **Fork risks:** There might be uncertainty around the correct fork choice when updating the EBO since it is not a fork oracle. For future verifiability efforts, it will likely be necessary to include blockhash in addition to the block number in the EBO’s output, making this even more important.
+
+# **High-Level Description**
+
+## Optimistic EBO Solution Considerations
+
+The goal of this solution is to decentralize the EBO while keeping the structure open for future modifications.
+
+Key improvements:
+
+- **Optimistic operators:** Implementing an optimistic oracle will allow EBO operations to be open and permissionless, improving decentralization, redundancy, and scalability.
+- **Timestamp comparison:** Instead of querying each RPC when detecting a new epoch, a delay is introduced to compare timestamps, reducing time sensitivity and improving the precision of fork selection.
+- **Escalating bond mechanism**: The modularity of the optimistic oracle allows for an escalating bond dispute system, reducing Arbitrator intervention and enabling lower initial stakes without compromising security.
+    - The Arbitrator multisig will act as the ultimate source of truth in the case of disputes in this initial phase, with potential future improvements.
+
+### Optimistic Design
+
+![image.png](../assets//gip-0080/Optimistic%20EBO.png)
+
+### **Optimistic operators**
+
+The optimistic approach allows off-chain agents to propose updates to the Optimistic EBO, which other participants can dispute by staking tokens or using alternative resolution mechanisms to align incentives and ensure accuracy.
+
+For this, we will use [Prophet](https://github.com/defi-wonderland/prophet-modules/tree/dev), an optimistic, modular, and public-good oracle that abstracts the complexity of the proposal and dispute process while offering the flexibility to customize its functionality as needed. For more details on Prophet, refer to the documentation [here](https://docs.prophet.tech/content/modules/index.html).
+~~~~
```

Comment:
Is this meant to be an HR? It seems to be causing subsequent text to render as code. Might need a blank line or another change.